src/BaGet.Web/Controllers/IndexController.cs

```diff
@@ -19,19 +19,20 @@ public object Get()
 
             // TODO: Client is picky about versions' formatting. Needs more research. See:
             // https://github.com/NuGet/NuGet.Client/blob/67d34ee8158597377fb26e06987ad6ad40a1b09b/src/NuGet.Core/NuGet.Protocol/Constants.cs
+            string version = "3.0.0";
             return new
             {
-                Version = "3.0.0-beta.1",
+                Version = version,
                 Resources = new[]
                 {
                     // Required
                     new ServiceResource("PackagePublish/Versioned", Url.PackagePublish()),
-                    new ServiceResource("SearchQueryService/Versioned", Url.PackageSearch()),
-                    new ServiceResource("RegistrationsBaseUrl/Versioned", Url.RegistrationsBase()),
-                    new ServiceResource("PackageBaseAddress/Versioned", Url.PackageBase()),
+                    new ServiceResource("SearchQueryService/" + version, Url.PackageSearch()),

```

Comment:
This breaks Visual Studio's search as `SearchQueryService/3.0.0` isn't a valid resource (the valid resources according to [the documentation](https://docs.microsoft.com/en-us/nuget/api/search-query-service-resource) are `SearchQueryService`, `SearchQueryService/3.0.0-beta`, `SearchQueryService/3.0.0-rc`).